Source/DafnyLanguageServer.Test/CodeActions/CodeActionsTest.cs

```diff
@@ -26,6 +26,23 @@ public class CodeActionTest : ClientBasedLanguageServerTest {
       return await client.ResolveCodeAction(codeAction, CancellationToken);
     }
 
+    [TestMethod]
+    public async Task CodeActionSuggestsRemovingAttribute() {
+      await TestCodeAction(@"
+method ><|>remove attribute|||{:dllimport}<| Foo()
+{ 
+}");
+    }
+
+    [TestMethod]
+    public async Task CodeActionSuggestsRemovingUnderscore() {
+      await TestCodeAction(@"
+method Foo()
+{
+  var ><|>remove underscore>>>x|||_x<| := 3; 
```

Comment:
I like your new language to encode code actions on top of a Dafny file, very clever. As you probably saw in the rest of the file, there was another language to do that, however, it did not have support yet for erasing things, only adding things.
Did you look into expanding that previous language? Your example could have look like this:
```suggestion
  var >>>[[remove underscore|x<<<_x]] := 3; 
```

Otherwise, if you want to keep the spirit of your language, I have a suggestion to increase its legibility
```suggestion
  var ><(>remove underscore|x<<<_x<) := 3; 
```
* Parentheses so that we visually group the entire code action.
* Similar to previous `[>` and `{>`
* The new separator after the label (`|`) does not need to be complicated since label probably won't contain that separator symbol (and if it does later, we could always change it later)
* Order of `<<<` indicates what is old (on the right of the arrow) vs. what is new (on the left of the arrow)
* Visually, `>` binds stronger than `|`, so now there is the label on one side and the transformation on the other side.

Reply:
> Parentheses so that we visually group the entire code action.

Good idea. Done

> Did you look into expanding that previous language?

I wanted to re-use the existing `MarkupTestFile` language, since that's also used outside of code actions. I think the `><` that we have there is good for indicating a position, so while the `>>>` makes more sense for code actions since the position is followed by an action, I think I'll keep the `><` for consistency.

> Order of <<< indicates what is old (on the right of the arrow) vs. what is new (on the left of the arrow)

Makes sense, but this operator is also the generic binding operator for metadata and code in `MarkupTestFile`. I've change dit from `|||` to `:::` since that seems more like a 'binding' operator.

---

If you want, I can also update the other tests to use this, arguably less nice language than the existing one.

Reply:
Nice, I find that it now reads nicely, especially when you can omit the position in front of the range.
Yes, I'd love that you update the other tests to use this _now arguably nice_ markup language, so that we use only one language. Easier to maintain.

Reply:
Done